app/src/main/res/layout/list_item_config.xml

```diff
@@ -17,21 +17,41 @@
         android:contentDescription="@string/content_description_provider_icon"
         android:src="@drawable/vpn_icon"/>
 
-    <TextView
-        android:id="@+id/profileName"
-        android:layout_width="wrap_content"
+    <LinearLayout
+        android:id="@+id/textContainer"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_centerVertical="true"
         android:layout_marginLeft="12dp"
         android:layout_marginRight="8dp"
         android:layout_toRightOf="@+id/providerIcon"
-        android:ellipsize="middle"
-        android:maxLines="1"
-        android:singleLine="true"
-        android:textColor="@android:color/black"
-        android:textSize="18sp"
-        tools:ignore="Deprecated"
-        tools:text="PROFILE: Secure Browsing"/>
+        android:orientation="vertical">
+
+        <TextView
+            android:id="@+id/profileName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="middle"
+            android:maxLines="1"
+            android:singleLine="true"
+            android:textColor="@android:color/black"
+            android:textSize="18sp"
+            tools:ignore="Deprecated"
+            tools:text="Secure Browsing"/>
+
+        <TextView
+            android:id="@+id/profileProvider"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:ellipsize="middle"
+            android:maxLines="1"
+            android:singleLine="true"
+            android:textColor="@android:color/black"
+            android:textSize="15sp"
+            tools:ignore="Deprecated"
+            tools:text="cloud.eduvpn.nl"/>
```

Comment:
An example text, which is only shown in the layout editor. It is discarded at build time. Useful for seeing the changes visually in the layout editor.